Python/alerts-to-discord/main.py

```diff
@@ -58,9 +60,10 @@ def post_message_to_discord(bot_name: str, message_body: str,
 # [START v2CrashlyticsAlertTrigger]
 @crashlytics_fn.on_new_fatal_issue_published(secrets=["DISCORD_WEBHOOK_URL"])
 def post_fatal_issue_to_discord(
-        event: crashlytics_fn.CrashlyticsNewFatalIssueEvent) -> None:
+    event: crashlytics_fn.CrashlyticsNewFatalIssueEvent,
+) -> None:
     """Publishes a message to Discord whenever a new Crashlytics fatal issue occurs."""
-# [END v2CrashlyticsAlertTrigger]
+    # [END v2CrashlyticsAlertTrigger]
```

Comment:
i think it needs to be col 0, throughout

Reply:
Per our conversation, looks like indented tags are ok